### PR TITLE
Visual triplets

### PR DESCRIPTION
**Pull request by umarmanzoor**

### CODE REVIEW

File: src/main/java/edu/tulane/cs/hetml/vision/CLEFImageReader.java [hunk 1]
```diff
@@ -68,13 +70,16 @@ public CLEFImageReader(String directory, String trainFilePath, String testFilePa
         testSegments = new ArrayList<>();
         testRelations = new ArrayList<>();
 
+        // all Segment
+        allSegments = new ArrayList<>();
+
         path = directory;
         // Load redefined segment relations
-        getRedefinedRelations(directory);
+//        getRedefinedRelations(directory);
```
Comment (Rahgooy) [inline src/main/java/edu/tulane/cs/hetml/vision/CLEFImageReader.java:78]: @umarmanzoor, We need to load relations, no?

Comment (umarmanzoor) [reply src/main/java/edu/tulane/cs/hetml/vision/CLEFImageReader.java:78]: No, these are redefined relations, we need the original ones

Comment (Rahgooy) [reply src/main/java/edu/tulane/cs/hetml/vision/CLEFImageReader.java:78]: And where we do load them?

Comment (umarmanzoor) [reply src/main/java/edu/tulane/cs/hetml/vision/CLEFImageReader.java:78]: getallImages(directory);

**Pull request merged**